src/cli/utils/flags_test.go

```diff
@@ -122,3 +122,26 @@ func (suite *FlagUnitSuite) TestAddS3BucketFlags() {
 	err := cmd.Execute()
 	require.NoError(t, err, clues.ToCore(err))
 }
+
+func (suite *FlagUnitSuite) TestAddFilesystemFlags() {
+	t := suite.T()
+
+	cmd := &cobra.Command{
+		Use: "test",
+		Run: func(cmd *cobra.Command, args []string) {
+			assert.Equal(t, "/tmp/test", flags.FilesystemPathFV, flags.FilesystemPathFN)
+			assert.Equal(t, true, flags.SucceedIfExistsFV, flags.SucceedIfExistsFN)
```

Comment:
nit: this can be shortened by using a more specific assertion
```suggestion
			assert.True(t, flags.SucceedIfExistsFV, flags.SucceedIfExistsFN)
```